hack/Dockerfile

```diff
@@ -141,13 +134,11 @@ RUN set -ex \
 		/src/src/dashboard/src/media \
 	' \
 	&& mkdir -p $internalDirs \
-	&& chown -R archivematica $internalDirs
+	&& chown -R archivematica $internalDirs \
+    && npm install -g npm@9.8.1 /src/src/dashboard/frontend
 
 USER archivematica
 
-RUN set -ex \
-	&& yarn --cwd=/src/src/dashboard/frontend install --frozen-lockfile
```

Comment:
If this is removed, should we also delete the `yarn.lock` file?

Reply:
Yes it is removed in next commit. Thanks for reminding me.